aep/general/0133/aep.md.j2

```diff
@@ -0,0 +1,246 @@
+# Create
+
+In REST APIs, it is customary to make a `POST` request to a collection's URI
+(for example, `/v1/publishers/{publisher}/books`) in order to create a new
+resource within that collection.
+
+Resource-oriented design (AEP-121) honors this pattern through the `Create`
+method. These RPCs accept the parent collection and the resource to create (and
+potentially some other parameters), and return the created resource.
+
+## Guidance
+
+APIs **should** generally provide a create method for resources unless it is
+not valuable for users to do so. The purpose of the create method is to create
+a new resource in an already-existing collection.
+
+Create methods are specified using the following pattern:
+
+- The response **must** be the resource itself. There is no separate response
+  schema.
+  - The response **should** include the fully-populated resource, and **must**
+    include any fields that were provided unless they are input only (see
+    AEP-203).
+  - If the create RPC is [long-running](#long-running-create), the response
```

Comment:
nit: I've never liked the fact that we have to restate long-running everywhere: it's pretty redundant. I kind of feel like we should at most just reference long-running methods along the lines of "this may be a long running method", and consolidate guidance to LROs there.

thoughts?

Reply:
Discussed live:  agreed on the approach, but we'll do a separate cleanup PR for this.  @toumorokoshi will file an issue.

Reply:
https://github.com/aep-dev/aep.dev/issues/106